src/main/java/com/hermesworld/ais/galapagos/topics/service/impl/TopicServiceImpl.java

```diff
@@ -51,7 +53,7 @@ public class TopicServiceImpl implements TopicService, InitPerCluster {
 
     private final GalapagosEventManager eventManager;
 
-    private final MessagesService messagesService;
+    private final MessagesServiceImpl messagesServiceImpl;
```

Comment:
Type used should still be "MessagesService", so we can change the implementation e.g. during Unit Tests. Spring will automatically choose MessagesServiceImpl as the correct implementation at runtime.